### DISCUSSION TITLE
On timeseries panels, add option to display axis on both the left and right

### DISCUSSION THREAD

**Discussion by jeffreymonroe** [grafana/grafana#59738] [Dashboards, Panels & Visualizations]
Currently, the only way to achieve this is to duplicate the query and use overrides to set the second query to display on the right.

![image](https://user-images.githubusercontent.com/7051807/205340816-bef60228-d3e1-4837-8395-1dd1ba856d6a.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
